Packs/GSuiteAdmin/ReleaseNotes/1_2_0.md

```diff
@@ -0,0 +1,15 @@
+#### Integrations
+##### Google Workspace Admin
+- Added 7 commands:
+    - ***gsuite-user-reset-password***
+    - ***gsuite-chromebrowserdevice-move-ou***
+    - ***gsuite-chromebrowserdevice-list***
+    - ***gsuite-policy-modify***
+    - ***gsuite-policy-schemas-list***
+    - ***gsuite-policy-resolve***
+    - ***gsuite-policy-groups-delete***
+- Added the parameter "Customer ID" and updated all commands that use customer_id as argument to not mandatory. All commands can use either argument or parameter.
+- Updated the commands gsuite-custom-user-schema-create and gsuite-custom-user-schema-update, now the argument 'field_json_entry_id' is getting an entry ID and will read the content of the json file in this entry id.
+- Updated the command gsuite-datatransfer-request-create, now the argument 'applications_raw_json_entry_id' is getting an entry ID and will read the content of the json file in this entry id.
+
+- Updated the Docker image to: *demisto/googleapi-python3:1.0.0.91615*.
```

Comment:
```suggestion
- Added the *Customer ID* parameter. For all commands that accept a *customer_id* argument, this argument is now optional. However, either the argument or the parameter must be provided.
- **Breaking changes:**
  - In the ***gsuite-custom-user-schema-create*** and ***gsuite-custom-user-schema-update*** commands, the expected *field_json_entry_id* argument value is an entry ID of a json file.
  - In the ***gsuite-datatransfer-request-create*** command, the expected *applications_raw_json_entry_id* argument value is an entry ID of a json file.
- Updated the Docker image to: *demisto/googleapi-python3:1.0.0.91615*.
```

Reply:
changing the breaking changes lines to comply with the doc template:

Release notes must start with one of the following prefixes:

'Added support for '
'Added the '
'Added a'
'Added an '
'Fixed an issue '
'Improved implementation '
'Updated the'
'You can now '
'Deprecated. '
'Deprecated the'
'Improved layout'
'Created a new layout'
'Playbook now supports'
'Created a new playbook'

Reply:
they can't start with "In the"